standalone/src/serial.rs

```diff
@@ -55,29 +55,32 @@ impl<S: Read + Write> Inner<S> {
     ) -> Result<Tag> {
         let _ = self.mctp.update(self.now());
         let cookie = None;
-        let r = self.mctpserial.send_fill(
+        let mut fragmenter = self.mctp.start_send(
```

Comment:
I'd consider revamping `standalone` to use `Router` and then serial.rs would be two simpler loops. That can be done later though.

Reply:
I was debating doing that, but settled on the direct stack approach as a minimum change. Happy to convert later if `Router` makes more sense in general.